docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md

```diff
@@ -0,0 +1,79 @@
+# 1. Use Case description
+
+## Name of use case
+
+Manual adding of data
+
+## 1.1 Brief Description
+
+Every user should have the possibility to visualize his cash flow in a diagram according to categories and depending on a period of time. The following data are visualized:
+
+- summed cash flow 
+per category depending on period
+
+
+# 2 Flow of Events
+
+## 2.1 Basic Flow
+
+- User clicks on dashboard
+- User selects a specific time slot
+  Data gets checked for availability depending on the time period
+- If more than one cash flow of a category is available, the value is summed up
+- The Dashboard will be updated
+
+### 2.1.1 Activity Diagram
+
+![show_dashboard_data Activity Diagram](show_dashboard.drawio.svg)
+
+### 2.1.2 Mock-up
+
+![dashboard](../UC_5_show_data_in_dashboard/dashboard.png)
+![dashboard](../UC_5_show_data_in_dashboard/dashboard_zeitraum.png)
+
+
+
+
+### 2.1.3 Narrative
+
+```gherkin
+Feature: show data in dashboard
+
+  As a signed-in user
+  I want to display the summed cash flows 
+  in a pie chart 
+  depending on a time window and category
+
+  Background:
+    And I am on the dashboard
+
+  Scenario: show cash flow data depending on time slot
+    Given I am signed in with username "USER" and password "PASSWORD"
+    And I am on the "dashboard" page
+    When I select a specific time slot
```

Comment:
Should we also add the scenario when the user does not select any specific timeslot?

In this case we should mentioned that the standart value is the complete time from the first to the last cashflow entry

Reply:
I think its a good idea